docstr_coverage/visitor.py

```diff
@@ -1,19 +1,29 @@
 """This module handles traversing abstract syntax trees to check for docstrings"""
+import re
+import tokenize
```

Comment:
The gouping I chose (more accurately: _the grouping my automatic code organizer chose_ ;-) ) is based on pep 8 style guide:

```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```

`re` and `tokenize` are standard libraries. `pytest` is a third party library, `docstr_coverage` is the local application import. 
I thus think the ordering follows standard convention - but I am happy to be convinced otherwise if I misunderstood something or if docstr_coverage follows is custom style.

Reply:
Thank you for your thoughtful organization. I was just referring to the import lines for `re`, `tokenize`, and `ast`, all of which are standard libraries. This file doesn't import `pytest` or `docstr_coverage` as `tests/test_coverage.py` does. I think we may be getting this comment mixed up with the one in that file.

Reply:
O. Yes. My bad, I confused the file with your other comment. :man_facepalming: Still. Afaik plain module imports (e.g. `import re`) should come before specific imports (e.g. `from ast import bla, blu, asd`). I'm quite sure about that as this is the case in all the example in pep 8 referred above and as if I try to reorder, my formatter gets angry at me :smile: 

This seems to be confirmed [in these non-official import guidelines](https://github.com/reddit-archive/reddit/wiki/PythonImportGuidelines).
I have to admit that I found the link [in this stackoverflow question of people having the same problem we have.](https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement)

I post this as it's quite funny to see that we're not the first ones getting deadlocked by this issue. However, as a maintainer, you should obviously have the last word about it and I'm fine with strict alphabetical order. If you prefer alphabetical order just tell me (one more time :man_facepalming: ) and I'll change it :-)

Reply:
Hahaha sorry for wasting your time with something this silly. You make a great point! Thanks for doing the research and walking me through it.

Reply:
For the record: No time wasted. I appreciate your work to ensure the code has a good quality.